### ISSUE TITLE
Task log

### ISSUE THREAD

**Issue by a7hybnj2** [GothenburgBitFactory/timewarrior#150]
Add functionality to the hook to track “task log start:### end:### description”

Use case: after a phone call I like to add it to the log including a start and stop time. Would be nice if this could also show up on the timew summary.

**Comment by a7hybnj2** [comment]
In my initial playing around with hooks I don't think on-modify fires on 'task log' it does appear to fire 'on-add' however.

**Comment by pbeckingham** [comment]
That makes sense, 'task log' is just 'task add' with an implicit 'status:completed'.

**Comment by lauft** [comment]
Does `task log` trigger any hook at all?
If not, we need to defer this issue to taskwarrior as well if such a feature is feasible.
In any case we need a new hook script which then triggers the appropriate timewarrior actions (I guess it just a simple `timew track <start> - <end>`).

**Comment by a7hybnj2** [comment]
Yes, 'task log' appears to fire 'on-add' as I mentioned previously. I haven't had the time to do much testing beyond that.
